### ISSUE TITLE
[FEATURE] Add field handle as tooltip to field labels

### ISSUE THREAD

**Issue by proimage** [craftcms/cms#5833]
### Description
When editing an entry, it would be useful to add simple, browser-native tooltips to field labels containing the handle of the field: `<label title="Handle: myCustomField">My Custom Field</label>`

This should probably only be done for admins / users who have permission to manage sections & fields.

**Comment by michtio** [comment]
@proimage I do like your idea, but I would even only introduce that when devMode is on :)

**Comment by brandonkelly** [comment]
Just added the ability to completely toggle field labels with their handles, when the <kbd>Option</kbd>/<kbd>ALT</kbd> key is pressed, for Craft 3.5.

![Some form fields revealing their handles in place of their lables](https://user-images.githubusercontent.com/47792/77599513-24e63980-6ec2-11ea-8648-619ddf905a8e.png)

**Comment by proimage** [comment]
Is that restricted to the site being in dev mode, or to users with field editing abilities? I imagine it might cause some minor confusion among content editors otherwise.

**Comment by brandonkelly** [comment]
@proimage it’s available regardless, but now that I think about it, I think I’ll make it limited to Admins.

**Comment by davidhellmann** [comment]
@brandonkelly nice, but now you can also add a edit link next when handles are visible to quick edit fields. use it very often. Can uninstall one plugin :)

**Comment by brandonkelly** [comment]
@davidhellmann can you post a new issue for that?

**Comment by remcoov** [comment]
Great feature! What would be practical in, in my opinion, is to have a 'copy icon' that you could click when holding the OPTION/ALT key, to quickly copy the handle name.

**Comment by proimage** [comment]
Having finally gotten to use this feature now, the Windows experience is possibly unintended behavior.

tl;dr: Should probably change the hotkey from ALT to CTRL.

In Windows, tapping the ALT key changes the focus to the menu bar (File, Edit, Window, Help, etc). This is for accessibility purposes/keyboard navigation of programs, etc, and is a standard convention. So if I'm in the Craft CP editing an entry, and say the Title field has focus. If I tap the ALT key to see a field handle, and try to continue typing, my focus will no longer be in the Title field of the browser, but on the browser's menu bar (for Chrome, it's the 3 vertical dots menu in the top-right that gets focus). At that point, no alphanumeric keys have any response—until the spacebar is pressed, which actually opens the menu. At that point, all the menu items have hotkey letters underlined, so pressing "N" will open a **N**ew window, while pressing "X" will E**x**it the browser and all its windows and tabs. O.O

CTRL, on the other hand, is a much more *neutral* modifier key, and by itself does nothing (unless the user has customized things through 3rd-party apps or whatever). What it *is* however is the first half of the Copy command (CTRL+C), which makes a lot more sense for Craft to hook into: Hold CTRL and the field handles show -> Mouse-select the desired handle -> Tap C (CTRL is still held) to copy the selected handle -> Release CTRL.
